lib/discordrb/data/channel.rb

```diff
@@ -698,6 +698,19 @@ def leave_group
 
     alias_method :leave, :leave_group
 
+    # Creates a webhook in this channel
+    # @param name [String] the default name of this webhook.
+    # @param avatar [String] the default avatar URL to give this hook
+    # @param reason [String] the reason why this happened, for the audit logs
```

Comment:
Consistent language with other `@param reason` docstrings:
```suggestion
    # @param reason [String] the reason for the webhook creation.
```